src/main/java/net/sf/jabref/logic/labelPattern/LabelPatternUtil.java

```diff
@@ -175,8 +175,11 @@ private static String unifyDiacritics(String content) {
      * @return True if the author or editor is an institution.
      */
     private static boolean isInstitution(String author) {
-        return (author.charAt(0) == '{')
-                && (author.charAt(author.length() - 1) == '}');
+        if (!(author.isEmpty())) {
+            return (author.charAt(0) == '{') && (author.charAt(author.length() - 1) == '}');
+        } else {
+            return false; // In case of empty or null author
```

Comment:
Yes, bad comment. I had null check earlier, but then I realized that the method is never(?) called with a null argument anyway. Let me double check and come back with the correct solution. :-)